core/src/main/java/ch/unibas/dmi/dbis/polyphenydb/statistic/StatisticsManager.java

```diff
@@ -0,0 +1,498 @@
+        if ( !map.containsKey( schema ) ) {
+            map.put( schema, new HashMap<>() );
+        }
+
+        if ( !map.get( schema ).containsKey( table ) ) {
+            map.get( schema ).put( table, new HashMap<>() );
+        }
+        map.get( schema ).get( table ).put( column, statisticColumn );
+
+    }
+
+
+    private void put( String schema, String table, String column, StatisticColumn statisticColumn ) {
+        put( this.statisticSchemaMap, schema, table, column, statisticColumn );
+
+    }
+
+
+    /**
+     * Method to get a generic Aggregate Stat
+     * null if no result is available
+     *
+     * @return a StatQueryColumn which contains the requested value
+     */
+    private StatisticQueryColumn getAggregateColumn( QueryColumn column, String aggregate ) {
+        return getAggregateColumn( column.getSchema(), column.getTable(), column.getName(), aggregate );
+    }
+
+
+    /**
+     * Queries the database with a aggreagate query
+     *
+     * @param aggregate the aggregate funciton to us
+     */
+    private StatisticQueryColumn getAggregateColumn( String schema, String table, String column, String aggregate ) {
+        String query = "SELECT " + aggregate + " (" + StatisticQueryProcessor.buildName( schema, table, column ) + ") FROM " + StatisticQueryProcessor.buildName( schema, table ) + getStatQueryLimit();
+        return this.sqlQueryInterface.selectOneStat( query );
+    }
+
+
+    /**
+     * Gets the configured amount + 1 of unique values per column
+     *
+     * @return the unique values
+     */
+    private StatisticQueryColumn getUniqueValues( QueryColumn column ) {
+        String tableName = StatisticQueryProcessor.buildName( column.getSchema(), column.getTable() );
+        String columnName = StatisticQueryProcessor.buildName( column.getSchema(), column.getTable(), column.getName() );
+        String query = "SELECT " + columnName + " FROM " + tableName + " GROUP BY " + columnName + getStatQueryLimit( 1 );
+        return this.sqlQueryInterface.selectOneStat( query );
+    }
+
+
+    /**
+     * Gets the amount of entries for a column
+     */
+    private Integer getCount( QueryColumn column ) {
+        String tableName = StatisticQueryProcessor.buildName( column.getSchema(), column.getTable() );
+        String columnName = StatisticQueryProcessor.buildName( column.getSchema(), column.getTable(), column.getName() );
+        String query = "SELECT COUNT(" + columnName + ") FROM " + tableName;
+        StatisticQueryColumn res = this.sqlQueryInterface.selectOneStat( query );
+
+        if ( res != null && res.getData() != null && res.getData().length != 0 ) {
+            try {
+                return Integer.parseInt( res.getData()[0] );
+            } catch ( NumberFormatException e ) {
+                log.error( "Count could not be parsed." );
```

Comment:
Please also include the Exception as well as the column name in the logging statement.
```suggestion
                log.error( "Count could not be parsed for column {}.", column.getQualifiedColumnName() e );
```